### PR TITLE
feat: explicit workflow invocation

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

**Review by metacosm — APPROVED** [review]

File: docs/documentation/workflows.md [hunk 1]
```diff
@@ -341,39 +341,30 @@ In other words if a Kubernetes Dependent Resource depends on another dependent r
...
-    implements Reconciler<WorkflowExplicitInvocationCustomResource> {
-    
-  @Override
-  public UpdateControl<WorkflowExplicitInvocationCustomResource> reconcile(
-      WorkflowExplicitInvocationCustomResource resource,
-      Context<WorkflowExplicitInvocationCustomResource> context) {
-    
-     // additional logic before the workflow is executed
-      
-     context.managedWorkflowAndDependentResourceContext().reconcileManagedWorkflow();
```
Comment (csviri) [inline docs/documentation/workflows.md]: @metacosm why did you delete sample code from here?

Comment (metacosm) [reply docs/documentation/workflows.md]: Because it doesn't add much, imo, and would need to be kept up to date (as are the other samples)…

Comment (csviri) [reply docs/documentation/workflows.md]: will merge this now as it is, when we migrate to docsy, would be nice to revisit the principles for the dox though
